aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -3975,6 +3979,11 @@ func createEdgeTransitGateway(d *schema.ResourceData, client *goaviatrix.Client,
 		}
 	}
 
+	// for AEP EAT gateway, set the eip_map, bgp polling time, bgp neighbor status polling time, local_as_number and prepend_as_path after the transit is created. AEP gateways take ~15 mins to be up and running. Updates to the gateway while is in the process of being created will fail.
+	if goaviatrix.IsCloudType(cloudType, goaviatrix.EDGENEO) {
+		return nil
+	}
+
```

Comment:
curious as to how returning nil here will cause those attributes to be set after transit is created? Is it meant to be set as part of the next read operation?